.github/labeler.yml

```diff
@@ -152,3 +152,9 @@
   - "dts/arm/silabs/**/*"
   - "dts/bindings/**/silabs,gecko*"
   - "soc/arm/silabs_exx32/**/*"
+"Bluetooth Controller":
```

Comment:
Move this alongwith bluetooth and bluetooth mesh at lines 98-102

Reply:
I wasn't sure what order was expected. I added them at the end so we'd have an order of a-z-A-Z, but we can move this in the "`area`" area.